helm/prometheus-rules/templates/alerting-rules/inhibit.prometheus-agent.rules.yml

```
@@ -0,0 +1,25 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: inhibit.prometheus-agent.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: inhibit.prometheus-agent
+    rules:
+    # this inhibition is firing once a cluster is not running prometheus-agent.
+    # we use `kube_namespace_created` to get the list of prometheus namespaces, and we extract the clusterID
+    # then we compare the list given from prometheuses namespaces with the `app_operator_app_info` 
+    # which is a metric for app deployment with filtering of  deployed observability-bundle.
```

I think we can simplify the explanations like that:

```suggestion
    # this inhibition fires when a cluster is not running prometheus-agent.
    # we retrieve the list of existing cluster IDs from `kube_namespace_created`
    # then compare it with the list of deployed observability-bundles from `app_operator_app_info`
```